32blit-sdl/CMakeLists.txt

```diff
@@ -15,9 +15,14 @@ if(EMSCRIPTEN)
 else()
 	find_package(SDL2 REQUIRED)
 
-	target_include_directories(BlitHalSDL
-		PRIVATE	${SDL2_INCLUDE_DIRS}
-	)
+	# if using cmake configuration files SDL_* is not defined but the SDL2::SDL2 target is
```

Comment:
Would be more accurate to say cmake generated configuration files here...

Reply:
Worth changing? Possibly also expanding the comment to say that this specifically addresses Visual Studio? Or does this apply to other scenarios too?

Reply:
It would apply to any system that has SDL built using its CMake build. SDL has two build systems and they generate different (and incompatible) CMake configurations...